packages/repository-json-schema/src/build-schema.ts

```diff
@@ -176,34 +160,40 @@ export function modelToJsonSchema(ctor: Function): JSONSchema {
     // populating "properties" key
     result.properties[p] = metaToJsonProperty(metaProperty);
 
+    // handling 'required' metadata
```

Comment:
Moved here from the bottom of the loop body to allow me to use early `continue`.